backend/src/main/java/gov/cdc/usds/simplereport/config/authorization/DemoAuthenticationConfiguration.java

```diff
@@ -136,5 +149,11 @@ public DemoAuthorizationService(OktaRepository oktaRepo, IdentitySupplier getCur
           _oktaRepo.getOrganizationRoleClaimsForUser(username);
       return claims.isEmpty() ? List.of() : List.of(claims.get());
     }
+
+    @Override
+    public boolean isSiteAdmin() {
+      final String userEmail = _getCurrentUser.get().getUsername().toLowerCase();
```

Comment:
Honestly the toLowerCase is probably an unnecessary luxury in the test/demo context, but it's harmless I suppose. 🙂